test/scripts/e2e_subs/v26/short-asset-url-only.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+date '+short-asset-url-only start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+ASSET_NAME='Birlot : décollage vs. ࠶🦪'
+
+set +o pipefail
+# longer than 32-byte ASA URLs should fail
+${gcmd} asset create \
+    --creator ${ACCOUNT} \
```

Comment:
"${ACCOUNT}"

Reply:
Yes, I noticed it as well but I didn't want to mess anything up so I copied it verbatim from the `asset-misc.sh` file. I'll update it.